go.mod

```
@@ -1,16 +1,16 @@
 module github.com/rook/kubectl-rook-ceph
 
-go 1.22.5
+go 1.23
```

```suggestion
go 1.23
```

let''s not change this one for now, we want to keep what it is [rook](https://github.com/rook/rook/blob/master/go.mod#L3)

---

done